ci/migrate/setup.sh

```diff
@@ -40,7 +40,9 @@ kubectl wait --for=condition=Ready pod -l lkey=lvalue --timeout=600s
 echo "Install cstor & csi operators"
 
 kubectl apply -f ./ci/migrate/migrationCRD.yaml
-kubectl apply -f https://raw.githubusercontent.com/openebs/charts/gh-pages/cstor-operator.yaml
+wget https://raw.githubusercontent.com/openebs/charts/gh-pages/cstor-operator.yaml
+sed -ie 's|value: "0"|value: "1"|g' cstor-operator.yaml
```

Comment:
Is there any other saw way to do this. What if other values also get changed this way..

Reply:
I have checked there is only one occurrence of `value: "0"` in the yaml for now. I not sure how to do it in in any other way.